skt/runner.py

```diff
@@ -173,7 +173,7 @@ def __getresults(self):
             logging.error('All test sets aborted or were cancelled!')
             return SKT_ERROR
 
-        for job, recipe_sets in self.job_to_recipe_set_map.items():
+        for recipe_sets in self.job_to_recipe_set_map.itervalues():
```

Comment:
I'd prefer to use unnamed variable (`_`) instead of this change - there is no `itervalues()` method in Python3 so while this change solves the unused variable issue, it complicates the issue about Python3 compatibility.

Reply:
I fix it